cmd/skaffold/app/flags/image.go

```diff
@@ -61,11 +56,11 @@ func (i *Images) Set(value string) error {
 	return nil
 }
 
-// Type Implements Type() method for pflag interface
-func (i *Images) Type() string {
-	return fmt.Sprintf("%T", i)
+func(i *Images) Replace([] string) error {
+
```

Comment:
@briandealwis I am not too sure how would this interface be implemented, would we provide an array of strings and then replace thme in the images array ?

Reply:
@sladyn98 I think so, Looks like you could basically loop over the passed in data and use the `convertImageToArtifact()` function to populate a new `[]image`